tests/jsx-text-wrap/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -474,23 +513,9 @@ x = (
 
 x = (
   <div>
-    before
-    {stuff}
-    after
-    {stuff}
-    after
-    {stuff}
-    after
+    before{stuff}after{stuff}after{stuff}after{stuff}after{stuff}after{stuff}
     {stuff}
-    after
-    {stuff}
-    after
-    {stuff}
-    {stuff}
-    {stuff}
-    after
-    {stuff}
-    after
+    {stuff}after{stuff}after
```

Comment:
Why is there one line with only a single `{stuff}` in it? This is a weird edge case so it doesn’t matter much, but I’d like to understand how things work.

Reply:
Because we have three jsx expressins one after another. The same [here](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAPABAXnQHgCYCWAbgHzByoAOATnAM50HQCMAvuVbQ01AEzsUa9RhADMrHAHpCpEABoQESjB51koAIbVqEAO4AFLQjUoNRCATzyQAI2oawAazgwAypQcEoAc2QxqAK5wCl50cNQw+vbeALYayABmGgA2YQoAVnSoAEL2Ti6uGjFwADJecIkpaSAe1GHUyLYaNgCeydDWNF4wAOqWMAAWyAAcAAwKNBBhPfaUjUL1RBUKtACOAQS0URqx8UhJqcEgYTEEfoFHjD7JcACKARDwlYcKMM19eIPIvK-2BMlebwAYQgMTijSg0GWIACYQAKs0TAcwqxWEA)

Reply:
We don't wrap text without space before/after jsx expression, but we wrap multiple jsx expressions, don't we?

Reply:
Ah, now I get it. Makes sense! :+1: